app/assets/javascripts/chf_ga_events.js

```diff
@@ -2,13 +2,15 @@
 // https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/assets/javascripts/hyrax/ga_events.js
 //
 // We add our own more flexible.
+// Can be used by a container or directly by a target; just requires the 'ga-event-data' class on the element holding the data.
 //
 // https://developers.google.com/analytics/devguides/collection/gajs/eventTrackerGuide
 
-$(document).on('click', '*[data-analytics-event]', function(e) {
+$(document).on('click', '.ga-event-data', function(e) {
   _gaq.push(['_trackEvent',
```

Comment:
Where possible/convenient (and it isn't always), I like using [data] attributes for javascript hooks, instead classes -- classes are for CSS styling, data attributes are for JS hooks, which are different things. 

Did you make this change intentionally, or can we go back to a data hook?  I liked naming the data hook generically after 'analytics' not 'ga' specifically, hypothetically it could be used for alternative analytic services too, but I feel less strongly about that if you think naming it ga is less confusing.